lib/utils/http-requestor.js

```diff
@@ -91,7 +91,7 @@ class HttpRequestor extends BaseRequestor {
     const url = hook.url || hook;
     const method = hook.method || 'POST';
     let buf = '';
-
+    httpHeaders = {['user-agent']: 'jambonz'};
```

Comment:
I think this should be controlled by the presence of an env var that optionally provides http headers to be used on callbacks in the form of a json-ified string

Reply:
JAMBONZ_WEB_HOOK_HEADER = {['user-agent']: 'jambonz'} something like this ill have it done

Reply:
just checking if you still are planning on this